src/mlagility/cli/report.py

```diff
@@ -1,521 +1,216 @@
 import os
 import csv
-import dataclasses
-from typing import List
-import yaml
+from datetime import datetime
+from pathlib import Path
+from dataclasses import dataclass
 import groqflow.common.printing as printing
 import groqflow.common.build as build
 import groqflow.common.cache as cache
-from groqflow import groqmodel
+from groqflow.groqmodel import groqmodel
+from mlagility.common import labels
+from mlagility.api.ortmodel import ORTModel
+from mlagility.api.trtmodel import TRTModel
+from mlagility.api.devices import BenchmarkException
+from mlagility.common.filesystem import check_cache_dir
+
+
+def _update_numeric_attribute(new_val, current_val, default="-"):
+    """
+    Updates a numeric attribute if needed
+    """
+    if current_val == default:
+        return new_val if new_val is not None else default
+    else:
+        return current_val
```

Comment:
If I'm reading this correctly, the `new_val` is silently discarded if it disagrees with `current_val`. I thought you were going to print a warning or something when that happened?

Reply:
Added warning!